react-native/react/tracker/proofs-render.desktop.js

```diff
@@ -4,32 +4,76 @@
 import React from '../base-react'
 import BaseComponent from '../base-component'
 
+// TODO const when integrating
+const verified = 'verified'
+const checking = 'checking'
+const deleted = 'deleted'
+const unreachable = 'unreachable'
+const pending = 'pending'
+
 export default class ProofsRender extends BaseComponent {
+  openLink (proof, platform) {
+    window.open(platform ? proof.platformLink : proof.proofLink)
+  }
+
+  metaColor (pp) {
+    return {
+      'new': 'orange',
+      deleted: 'red',
+      unreachable: 'red',
+      pending: 'gray'
+    }[pp.proof.meta]
+  }
+
+  tempStatus (pp) {
+    return {
+      verified: '[v]',
+      checking: '[c]',
+      deleted: '[d]',
+      unreachable: '[u]',
+      pending: '[p]'
+    }[pp.proof.status]
+  }
+
+  renderPlatformProof (pp) {
+    const name = pp.platform.name === 'web' ? pp.platform.uri : pp.platform.name
+    console.log(name)
+    return (
+      <div style={{display: 'flex'}}>
+        <p title={name} style={{width: 40, marginRight: 10, cursor: 'pointer'}} onTouchTap={() => this.openLink(pp.platform.uri)}>{pp.platform.icon}</p>
+        <div style={{display: 'flex', flexDirection: 'column', alignItems: 'flex-start'}}>
+          <p style={{textDecoration: pp.proof.status === deleted ? 'line-through' : 'inherit', marginBottom: 0, cursor: 'pointer'}} onTouchTap={() => this.openLink(pp.platform.uri)}> {name}</p>
+          <span style={{backgroundColor: this.metaColor(pp)}}>{pp.proof.meta}</span>
+        </div>
+        <div style={{display: 'flex', flex: 1, justifyContent: 'flex-end', paddingRight: 20}}>
+          <p style={{cursor: 'pointer'}} onTouchTap={() => this.openLink(pp.proof.uri)}>{this.tempStatus(pp)}</p>
+        </div>
+      </div>
+    )
+  }
+
   render () {
     return (
-      <div style={{backgroundColor: 'green', display: 'flex', flex: 1, flexDirection: 'column', overflowY: 'auto'}}>
-        <h1>Proof1</h1>
-        <h1>Proof2</h1>
-        <h1>Proof3</h1>
-        <h1>Proof4</h1>
-        <h1>Proof5</h1>
-        <h1>Proof6</h1>
-        <h1>Proof7</h1>
-        <h1>Proof8</h1>
-        <h1>Proof9</h1>
-        <h1>Proof10</h1>
-        <h1>Proof11</h1>
-        <h1>Proof12</h1>
-        <h1>Proof13</h1>
-        <h1>Proof14</h1>
-        <h1>Proof15</h1>
-        <h1>Proof16</h1>
-        <h1>Proof17</h1>
-        <h1>Proof18</h1>
-        <h1>Proof19</h1>
-        <h1>Proof20</h1>
-        <h1>Proof21</h1>
+      <div style={{display: 'flex', flex: 1, flexDirection: 'column', overflowY: 'auto'}}>
+        { this.props.platformProofs && this.props.platformProofs.map(platformProof => this.renderPlatformProof(platformProof)) }
       </div>
     )
   }
 }
+
+ProofsRender.propTypes = {
+  platformProofs: React.PropTypes.arrayOf(React.PropTypes.shape({
+    platform: React.PropTypes.shape({
+      icon: React.PropTypes.string.isRequired,
+      name: React.PropTypes.string,
```

Comment:
if its a website the name isn't really a thing, other than the uri. we could say it has to be duped there...